package.json

```diff
@@ -6,7 +6,7 @@
     "start": "npm-run-all --sequential copy server",
     "copy": "node prepare.js",
     "server": "node server.js",
-    "test": "jest ./src/test --runInBand --detectOpenHandles",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js ./src/test --runInBand --detectOpenHandles",
```

Comment:
do we need `--experimental-vm-modules` and `babel-jest`? I would have thought that babel was transpiling away the modules

Reply:
I never figured out how to get `babel-jest` setup correctly so I have removed it and I am using `--experimental-vm-modules` with node, I do feel like I lack knowledge in this area so any help on that setup would be appreciated.